### ISSUE TITLE
cannot import name 'full' from 'scipy' (scipy 1.12.0, full  has been removed with this version)

### ISSUE THREAD

**Issue by Thomas-Ulrich** [SeisSol/PSpaMM#14]
Installing seissol with (on heisenbug):
```
CXX=g++ CC=gcc cmake -DNUMA_AWARE_PINNING=ON -DASAGI=ON -DCMAKE_BUILD_TYPE=Release -DHOST_ARCH=hsw -DPRECISION=double -DORDER=4 -DGEMM_TOOLS_LIST=LIBXSMM,PSpaMM .. 
```
Error log
```
projectDerivativeToNodalBoundaryRotated (16 AST(s))
subTriangleDisplacement (4 AST(s))
subTriangleVelocity (4 AST(s))
addVelocity (4 AST(s))
Generating kernels...
Calling external code generators...
Traceback (most recent call last):
  File "/import/tegern-data/ulrich/MyLibs/mambaforge3/bin/pspamm-generator", line 5, in <module>
    from pspamm.pspamm import main
  File "/import/exception-dump/ulrich/myLibs/spack-packages/linux-debian11-zen2/gcc-12.2.0/py-pspamm-develop-zwccp4lcotug5pl244tnmo3ftfzzxmix/pspamm/pspamm.py", line 7, in <module>
    from pspamm.matmul import *
  File "/import/exception-dump/ulrich/myLibs/spack-packages/linux-debian11-zen2/gcc-12.2.0/py-pspamm-develop-zwccp4lcotug5pl244tnmo3ftfzzxmix/pspamm/matmul.py", line 13, in <module>
    from pspamm.cursors import *
  File "/import/exception-dump/ulrich/myLibs/spack-packages/linux-debian11-zen2/gcc-12.2.0/py-pspamm-develop-zwccp4lcotug5pl244tnmo3ftfzzxmix/pspamm/cursors/__init__.py", line 1, in <module>
    from pspamm.cursors.matrix import Matrix
  File "/import/exception-dump/ulrich/myLibs/spack-packages/linux-debian11-zen2/gcc-12.2.0/py-pspamm-develop-zwccp4lcotug5pl244tnmo3ftfzzxmix/pspamm/cursors/matrix.py", line 8, in <module>
    from scipy import full, matrix # type: ignore
ImportError: cannot import name 'full' from 'scipy' (/import/tegern-data/ulrich/MyLibs/mambaforge3/lib/python3.10/site-packages/scipy/__init__.py)
Traceback (most recent call last):
  File "/export/dump/ulrich/trash/SeisSol/generated_code/generate.py", line 182, in <module>
    generator.generate(outputDir=cmdLineArgs.outputDir,
  File "/export/dump/ulrich/trash/SeisSol/generated_code/yateto/generator.py", line 378, in generate
    cache.generate(header, fRoutines.cpp, fGpulikeRoutines.cpp)
  File "/export/dump/ulrich/trash/SeisSol/generated_code/yateto/codegen/cache.py", line 39, in generate
    declaration = generator(name, cppFileName)
  File "/export/dump/ulrich/trash/SeisSol/generated_code/yateto/codegen/gemm/gemmgen.py", line 323, in __call__
    self._callGenerator(argList)
  File "/export/dump/ulrich/trash/SeisSol/generated_code/yateto/codegen/gemm/gemmgen.py", line 231, in _callGenerator
    raise RuntimeError(f"""GEMM code generator executable "{self._cmd}" failed. Thus, the kernel generation may be incomplete.
RuntimeError: GEMM code generator executable "/import/tegern-data/ulrich/MyLibs/mambaforge3/bin/pspamm-generator" failed. Thus, the kernel generation may be incomplete.
Given command: /import/tegern-data/ulrich/MyLibs/mambaforge3/bin/pspamm-generator 20 1 20 20 20 20 generic 0 --arch hsw --prefetching pfsigonly --output_funcname pspamm_m20_n1_k20_ldA20_ldB20_ldC20_alphageneric_beta0_alignedA1_alignedC1_transAFalse_transBFalse_pfsigonly --output_filename /export/dump/ulrich/trash/SeisSol/build/src/generated_code/subroutine.cpp --precision D
```

EDIT: works with scipy 1.11.0

**Comment by davschneller** [comment]
It should be fixed by #11 — but there's no new version yet; we can make one though now

**Comment by davschneller** [comment]
Is this issue still prevalent with v0.2.1? (after #11 has been merged?)
